docs/loading/Flink-connector-starrocks.md

```diff
@@ -1,173 +1,647 @@
 # Continuously load data from Apache Flink®
 
-import InsertPrivNote from '../assets/commonMarkdown/insertPrivNote.md'
+StarRocks provides a self-developed connector named Flink Connector for Apache Flink® (Flink connector for short) to help you load data into a StarRocks table by using Flink. The basic principle is to accumulate the data and then load it all at a time into StarRocks through [STREAM LOAD](../sql-reference/sql-statements/data-manipulation/STREAM%20LOAD).
+The Flink connector supports DataStream API, Table API & SQL, and Python API. The Flink connector also has a higher performance than [flink-connector-jdbc](https://nightlies.apache.org/flink/flink-docs-master/docs/connectors/table/jdbc/) provided by Apache Flink®.
 
-This topic describes how to load data from Apache Flink® to StarRocks.
+## Version requirements
 
-<InsertPrivNote />
+| Connector | Flink                    | StarRocks     | Java | Scala     |
+|-----------|--------------------------|---------------| ---- |-----------|
+| 1.2.7     | 1.11,1.12,1.13,1.14,1.15 | 2.1 and later | 8    | 2.11,2.12 |
 
-## Overview
+## Obtain Flink connector
 
-The flink-connector-jdbc tool provided by Apache Flink® may not meet your performance requirements in certain scenarios. Therefore we provide a new connector named flink-connector-starrocks, which can cache data and then load data at a time by using [Stream Load](./StreamLoad.md).
+You can obtain the Flink connector JAR file in the following ways:
```

Comment:
![image](https://github.com/StarRocks/starrocks/assets/57167462/e44a8b72-3070-4039-a851-42791ca3086f)
"Flink connector" or "Flink Connector"? Adhere to one usage throughout the doc.